Extension/.vscode/launch.json

```diff
@@ -135,9 +62,54 @@
                 "<node_internals>/**"
             ],
             "outFiles": [
-                "${workspaceFolder}/**/out/**/*.js",
+                "${workspaceFolder}/dist/**",
                 "!**/node_modules/**"
             ]
         }
+    ],
+    "inputs": [
+        {
+            "type": "pickString",
+            "id": "pickScenario",
+            "description": "Select which scenario to debug VSCode tests.",
+            "options": [
+                {
+                    "label": "MultirootDeadlockTest   ",
+                    "value": "${workspaceFolder}/test/scenarios/MultirootDeadlockTest/assets/test.code-workspace"
+                },
+                {
+                    "label": "SimpleCppProject   ",
+                    "value": "${workspaceFolder}/test/scenarios/SimpleCppProject/assets/simpleCppProject.code-workspace"
+                },
+                {
+                    "label": "SingleRootProject   ",
+                    "value": "${workspaceFolder}/test/scenarios/SingleRootProject/assets/"
+                }
+            ]
+        },
+        {
+            "type": "pickString",
+            "id": "pickWorkspace",
+            "description": "Select which workspace scenario to debug VSCode.",
+            "default": "-n",
+            "options": [
+                {
+                    "label": "(Debug without Workspace)   ",
```

Comment:
This text doesn't appear to be accurate -- it does a Debug with the previously open workspace (maybe without a workspace if that's what the previous state was).

Reply:
I can change that to "Debug with new window" which is what `-n` means. If it uses the last opened workspace, all the better.